saml-core/src/main/java/org/keycloak/saml/common/util/StaxParserUtil.java

```diff
@@ -411,7 +513,7 @@ public static EndElement getNextEndElement(XMLEventReader xmlEventReader) throws
      *
      * @return
      */
-    public static String getStartElementName(StartElement startElement) {
+    public static String getElementName(StartElement startElement) {
```

Comment:
This is just slight API change which is not customer facing, so we can leave it.